course.js

```diff
@@ -274,7 +274,7 @@ CourseJS.TimeSet = class TimeSet {
     /**
      * Gets a TBA object
      */
-    get TBA () {
+    getTBA () {
```

Comment:
This wasn't a typo; the space between get and TBA was meant to create a getter function for the class. Now, whether or not the getter is necessary is a different story.